src/edit_item.py

```diff
@@ -94,7 +94,7 @@ def apply(self):
         clipboard.add(self.props.application)
         graphs.refresh(self.props.application)
         if set_limits:
-            plotting_tools.optimize_limits(self)
+            plotting_tools.optimize_limits(self.props.application)
```

Comment:
This was just missed when moving from the seperate parent to this build in properties.